test/boost/reader_concurrency_semaphore_test.cc

```diff
@@ -742,7 +742,7 @@ SEASTAR_THREAD_TEST_CASE(test_reader_concurrency_semaphore_admission) {
 
     // inactive readers
     {
-        auto permit = semaphore.obtain_permit(schema, get_name(), 1024, db::timeout_clock::now(), {}).get();
+        auto permit = semaphore.obtain_permit(schema, get_name(), 1024, db::timeout_clock::now() + 99s, {}).get();
```

Comment:
probably this "magic number" deserves a comment.

Reply:
I replaced this with `db::no_timeout`.